### PR TITLE
Add spider for Chuy's

### LINKED ISSUE #3501: Chuy's
https://www.chuys.com/locations

### PR DESCRIPTION
**Pull request by mackerel10**
Closes #3501

### CODE REVIEW

File: locations/spiders/chuys.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                    "city": locality.split(", ")[0],
+                    "state": locality.split(", ")[1].split()[0],
+                    "postcode": locality.split(", ")[1].split()[1],
+                    "phone": phone,
+                    "website": website,
+                    "lat": lat,
+                    "lon": lon,
+                }
+            except Exception as e:
+                continue
```
Comment (iandees) [inline locations/spiders/chuys.py:51]: Instead of catching Exception and continuing, please let the exception out so we can fix it.

**Comment by iandees** [comment]
The output has some extra tabs in the address, can you remove those?

```
"addr:full": "10325 N. Michigan Rd.\t\t\t"
```

File: locations/spiders/chuys.py (hunk 1 above)
Comment (mackerel10) [reply locations/spiders/chuys.py]: Updated to print a statement and raise the exception

Comment (iandees) [reply locations/spiders/chuys.py]: No need to print. Just let the exception get raised. Remove the try/except.

Comment (mackerel10) [reply locations/spiders/chuys.py]: Done

**Pull request merged**